packages/tinacms/src/plugins/fields/DateFieldPlugin.tsx

```diff
@@ -57,6 +58,7 @@ export const DateField = wrapFieldsWithMeta<InputProps, DatetimepickerProps>(
             onFocus={input.onFocus}
             onChange={input.onChange}
             open={isOpen}
+            timeFormat={false}
```

Comment:
Why default this to false?

Reply:
It's a bit of a workaround. 
[react-datetime](https://github.com/YouCanBookMe/react-datetime) uses a separate dateformat and a timeformat, but when we [export the date value](https://github.com/tinacms/tinacms/blob/1d168ee01a774e1a58d4d7868c1161b13ea2d7a5/packages/tinacms/src/plugins/fields/dateFormat.ts#L36), we need a format with date & time combined.

The timepicker & non-utc-timezones also can cause issues (https://github.com/tinacms/tinacms/issues/257) with the picker, so I thought disabling it by default would simplify things for the majority of use-cases.